
cuda:0
Traceback (most recent call last):
  File "/home/ericji3/ECE496/ECE496/src/eval_model.py", line 55, in <module>
    main()
  File "/home/ericji3/ECE496/ECE496/src/eval_model.py", line 19, in main
    test_loader = make_dataset.import_testsets()
  File "/home/ericji3/ECE496/ECE496/src/data/make_dataset.py", line 53, in import_testsets
    test_loader[0] = DataLoader(dataset=test_dataset0, batch_size=16, shuffle=False, num_workers=6)
TypeError: float() argument must be a string or a number, not 'DataLoader'